### PR TITLE
update travis.yml to skip integration tests.

### PR DESCRIPTION
**Pull request by kmschoep-usgs**

### CODE REVIEW

File: .coveralls.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
+repo_token: XYOAjnCJSsJ7z9SKmKE3lN9TQdpKZwdd6
```
Comment (mbucknell) [inline .coveralls.yml]: Why is this needed? From what I read it's only needed if the repo is private

Comment (kmschoep-usgs) [reply .coveralls.yml]: We're throwing spaghetti at the wall to see what sticks.  This didn't work, so I'll remove it.
